docs/user-guide/server-side-install-overview.md

```diff
@@ -1,19 +1,124 @@
 # Server-side installation overview
 
-## Work items
+Review this overview to learn about the options to install Zowe server-side components, and the roles and responsibilities to perform this installation.
 
-To explain the server-side scope, flow, roles, and so on.
+* [Zowe server-side components in the installation](#zowe-server-side-components-in-the-installation)
+* [Installation of Zowe runtime](#installation-of-zowe-runtime)
+* [Zowe runtime installation methods and roadmap](#zowe-runtime-installation-methods-and-roadmap)
+    * [Details of runtime installation methods ](#details-of-runtime-installation-methods)
+* [Roles and responsibilities for server-side component installation](#roles-and-responsibilities-for-server-side-component-installation)
+    * [Tasks performed by the security administrator](#tasks-performed-by-the-security-administrator)
+    * [Tasks performed by the storage administrator](#tasks-performed-by-the-storage-administrator)
+    * [Tasks performed by the network administrator](#tasks-performed-by-the-network-administrator)
+    * [Tasks performed by the system programmar](#tasks-performed-by-the-system-programmar)
 
-What to be covered in the overview:
+## Complete installation ecosystem
 
-End-to-end install flow diagram 
+The following diagram presents the full ecosytem of options that can be used to install Zowe server-side components.
+![4-installation-methods](../images/install/4-installation-methods.png)
 
-config flow diagram
+## Zowe server-side components in the installation
 
-Roles & Responsibilities
+The following Zowe server-side components are included in this installation: 
 
-Zowe Key Terms & Concepts: Zowe runtime, Zowe Application Framework, Zowe API Mediation Layer, Z Secure Services (ZSS), Zowe Cross Memory Server (@Joe)
+* **Zowe Application Framework**  
+A web user interface (UI) that provides a virtual desktop containing several apps allowing access to z/OS functions.  <details><summary>Click here for more information about Zowe Application Framework.</summary> Zowe™ Application Framework includes apps for traditional access such as a 3270 terminal and a VT Terminal, as well as an editor and explorers for working with JES, MVS Data Sets, and Unix System Services.
+</details>
 
-Technology Needs: YAML, node.js, USS background details (existing content)
+*  **Zowe API Mediation Layer (API ML)**  
+Zowe API Mediation Layer provides a reverse proxy and enables REST APIs by providing a single point of access for mainframe service REST APIs.<details><summary>Click here for more information about API Mediation Layer.</summary> Such APIs include MVS Data Sets, JES, as well as working with z/OSMF. The API ML has dynamic discovery capability for these services and Gateway is also responsible for generating the authentication token used to provide single sign-on (SSO) functionality.</details>
```

Comment:
Don't mention MVS Data Sets and JES, just working with z/OSMF. 

Also instead of The API ML has dynamic discovery capability ...
rather use:
The API ML has a discovery service allowing for dynamic onboarding of the services. The Gateway is also responsible for authentication and provisioning of Single Sign On functionality across onboarded services.